.github/workflows/release-postinstall.js

```diff
@@ -150,27 +150,54 @@ var copyPlatformBinaries = (platformPath) => {
 };
 
 try {
-  fs.mkdirSync('_export');
+  fs.mkdirSync("_export");
 } catch (e) {
-  console.log('Could not create _export folder');
+  console.log("Could not create _export folder");
 }
 
 switch (platform) {
-  case 'win32':
-    if (arch() !== 'x64') {
-      console.warn('error: x86 is currently not supported on Windows');
+  case "win32":
+    if (arch() !== "x64") {
+      console.warn("error: x86 is currently not supported on Windows");
       process.exit(1);
     }
 
-    copyPlatformBinaries('windows-x64');
+    copyPlatformBinaries("windows-x64");
     break;
-  case 'linux':
-  case 'darwin':
+  case "linux":
+  case "darwin":
     copyPlatformBinaries(platform);
     break;
   default:
-    console.warn('error: no release built for the ' + platform + ' platform');
+    console.warn("error: no release built for the " + platform + " platform");
     process.exit(1);
 }
 
-require('./esyInstallRelease');
+// By now, the binaries should all be located under `bin/`.
+// TODO cleanup the code below
+
+copyBinary("bin/tailwind-ppx", "ppx");
+copyBinary("bin/use-tailwind-ppx", "use-ppx");
+
+function copyBinary(filename, destFilename) {
+  if (!fs.existsSync(destFilename)) {
+    copyFileSync(filename, destFilename);
+    fs.chmodSync(destFilename, 0755);
+  }
+
+  var destFilenameExe = destFilename + ".exe";
+  if (!fs.existsSync(destFilenameExe)) {
+    copyFileSync(filename, destFilenameExe);
+    fs.chmodSync(destFilenameExe, 0755);
+  }
+}
+
+function copyFileSync(source, dest) {
+  if (typeof fs.copyFileSync === "function") {
+    fs.copyFileSync(source, dest);
+  } else {
+    fs.writeFileSync(dest, fs.readFileSync(source));
+  }
+}
```

Comment:
This is the only meaningful part of this diff. In a later PR I'll fix all the formatting so we're on the same page.